api-contract/src/test/java/uk/gov/hmcts/payment/api/contract/FeeDtoTest.java

```diff
@@ -3,12 +3,14 @@
 import org.junit.Before;
 import org.junit.Test;
 
-import javax.validation.ConstraintViolation;
-import javax.validation.Validation;
-import javax.validation.Validator;
-import javax.validation.ValidatorFactory;
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import jakarta.validation.ValidatorFactory;
 import java.util.Iterator;
 import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
```

Comment:
These two can be removed now as they are not used in this file anymore. And `Iterator` on line 10, once you remove useless assignment on line 44